src/formatting/dateTime.ts

```diff
@@ -0,0 +1,11 @@
+export function formatDateTime (dateTime: Date, options: Intl.DateTimeFormatOptions = {}): string {
+  return dateTime.toLocaleString('en-US', options)
```

Comment:
please add another function here that returns the locale so that we can extend it and make it dynamic later without refactoring, for example:

```ts
export function currentLanguage(): string {
  return 'en-US'
}
```